### PR TITLE
Added ARTICLE

### PR DESCRIPTION
**Pull request by Abelkrijgtalles**
## Adding a language

- [x] The code displays "Hello World"
- [x] I have updated the readme to include the new language
- [x] I have incremented the language count in the readme

#### Link to programming language: 
https://github.com/SoicBRcode/article-lang

### CODE REVIEW

File: readme.md [hunk 1]
```diff
@@ -8,8 +8,7 @@ Thanks to everyone who continues to contribute; new languages are created every
 Make sure to see [contributing.md](/contributing.md) for instructions on contributing to the project!
 
 <!--Languages start-->
-
```
Comment (MrBrain295) [inline readme.md:11]: Can you replace the line that was removed.

Comment (Abelkrijgtalles) [reply readme.md:11]: What do you mean? You mean that I need to put line 10 on line 11?

**Comment by Abelkrijgtalles** [comment]
Ignore the review

**Comment by Abelkrijgtalles** [comment]
I've fixed the README.md

File: readme.md (hunk 1 above)
Comment (Abelkrijgtalles) [reply readme.md:11]: I've done it

**Pull request merged**